datastruc_ex7.py

```diff
@@ -0,0 +1,79 @@
+import random as rnd
+
+# ex7
+# a
+
+
+def func(n: int, random: bool, lst: list):
+    with open('WordData.txt', 'r') as file:
+        allText = file.read()
+        if random:
+            text_list = allText.split('\n')
+            randomlist = rnd.sample(range(0, len(text_list)), n)
+            selected_words = []
+            for i in range(len(randomlist)):
+                selected_words.append(text_list[randomlist[i]])
+            return selected_words
+        else:
+            text_list = allText.split('\n')
+            selected_words = []
+            for i in range(len(lst)):
+                selected_words.append(text_list[lst[i]])
+            return selected_words
```

Comment:
A lot of code here repeats itself in this section. for example:
line 22 and line 16 are identical.
13 and 19.
hint: 14-15 and 20-21 are ALMOST identical, you can also improve this.
REDUCE code that repeats itself.

Reply:
done.